tests/cucumber/features/channels/channel_creation.feature

```diff
@@ -0,0 +1,29 @@
+Feature: Channel creation
+  As an authenticated user
+  I want to create a channel
+
+  Background:
+    Given I am signed in
+    And I am on the new channels page
```

Comment:
When I tried this back when we were first setting up authentication, i could never get it to work. Can someone explain why it works now?